frontend-react/src/resources/ReportResource.ts

```diff
@@ -20,9 +20,36 @@ export default class ReportResource extends AuthResource {
     readonly fileName: string = "";
     readonly mimeType: string = "";
 
+    constructor(
+        reportId?: string,
+        sent?: number,
```

Comment:
This is just so we can create dummy items. There are no constructors in the parent class tree, so we're good to override.